doc/enterprise/auth/saml/saml_setup.md

```diff
@@ -0,0 +1,244 @@
+# Configure a SAML application
+
+Pachyderm supports Security Assertion Markup Language
+(SAML) providers to authenticate with Pachyderm which enables
+you to configure Single Sign-On (SSO) by using browser tokens.
+
+To complete the steps in this section, use the Minikube
+configuration with enabled port-forwarding. Many of the SAML
+service provider URLs below are set to a variation of localhost
+that only works if you are using port forwarding and your browser
+can access Pachyderm through localhost on the port forwarder’s usual ports.
+
+Because the examples in this section describe the configuration that
+uses Okta as an Identity Provider, you need to have Okta installed
```

Comment:
I would leave out "installed" (as Okta is a web service, so you typically configure it but don't typically install it)